app/src/main/java/com/github/se/polyfit/MainActivity.kt

```diff
@@ -42,6 +43,7 @@ class MainActivity : ComponentActivity() {
         val navController = rememberNavController()
         val navigation = Navigation(navController)
         NavHost(navController = navController, startDestination = Route.Register) {
+          composable(Route.Graph) { FullGraphScreen() }
```

Comment:
here is where we should pass in a back navigate to be used in the graph screen
```suggestion
          composable(Route.Graph) { FullGraphScreen(navigateBack = navigation::....) }
```

Reply:
Yes this will be done afterward I just wanted to finish this screen before linking it.